api/models/Payment.js

```diff
@@ -17,6 +17,10 @@ module.exports = (sequelize) => {
             type: DataTypes.INTEGER,
             allowNull: false
         },
+        currency: {
+            type: DataTypes.STRING,
+            allowNull: false
```

Comment:
For eventual production migration, this will fail because existing payments will not have a currency

We should either allow null values or have a separate migration that looks at every payment, finds the currency through the Client and apply it to the payments column